<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LCA of Biochar</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>165</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Create Blank Project</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Load Saved Project</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Location for Results</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Save Current Project</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="inputMask">
           <string>NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Create</string>
          </property>
          <property name="icon">
           <iconset resource="../appMainGui.qrc">
            <normaloff>:/icon/icon/create.png</normaloff>:/icon/icon/create.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Load</string>
          </property>
          <property name="icon">
           <iconset resource="../appMainGui.qrc">
            <normaloff>:/icon/icon/load.png</normaloff>:/icon/icon/load.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton">
          <property name="text">
           <string>Browse   </string>
          </property>
          <property name="icon">
           <iconset resource="../appMainGui.qrc">
            <normaloff>:/icon/icon/browse.png</normaloff>:/icon/icon/browse.png</iconset>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Save</string>
          </property>
          <property name="icon">
           <iconset resource="../appMainGui.qrc">
            <normaloff>:/icon/icon/save.png</normaloff>:/icon/icon/save.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>164</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QMenu" name="menuAnalysis">
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="title">
     <string>Analysis</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="actionMass_Balance"/>
    <addaction name="actionExergy_Analysis"/>
    <addaction name="actionEnergy_Analysis"/>
    <addaction name="actionImpact_Assesment"/>
   </widget>
   <addaction name="menuAnalysis"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMass_Balance">
   <property name="icon">
    <iconset resource="../appMainGui.qrc">
     <normaloff>:/icon/icon/mass balance.png</normaloff>:/icon/icon/mass balance.png</iconset>
   </property>
   <property name="text">
    <string>Mass Balance</string>
   </property>
  </action>
  <action name="actionExergy_Analysis">
   <property name="icon">
    <iconset resource="../appMainGui.qrc">
     <normaloff>:/icon/icon/exergy.png</normaloff>:/icon/icon/exergy.png</iconset>
   </property>
   <property name="text">
    <string>Exergy Analysis</string>
   </property>
  </action>
  <action name="actionEnergy_Analysis">
   <property name="icon">
    <iconset resource="../appMainGui.qrc">
     <normaloff>:/icon/icon/energy.png</normaloff>:/icon/icon/energy.png</iconset>
   </property>
   <property name="text">
    <string>Energy Analysis</string>
   </property>
  </action>
  <action name="actionImpact_Assesment">
   <property name="icon">
    <iconset resource="../appMainGui.qrc">
     <normaloff>:/icon/icon/Analysis.png</normaloff>:/icon/icon/Analysis.png</iconset>
   </property>
   <property name="text">
    <string>Impact Assesment</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../appMainGui.qrc"/>
 </resources>
 <connections/>
</ui>
